website/docs/components/app-header/partials/guidelines/overview.md

```diff
@@ -1 +1,7 @@
+!!! Warning
+
+We currently do not recommend using the `AppHeader` component in production.
```

Comment:
Small nit here, feel free to disregard if this feels unnecessary. Even though the page is hidden, should we actually say the `AppHeader` is not available to be used in production, which is a bit more true.

Reply:
I'm more opinionated about this now actually, maybe @didoo could chime in. Even though these components are hidden from the navigation, they are still searchable through Algolia. Is it worth it to try to exclude them?

Reply:
I updated the language at least for now. @jorytindall